great_expectations/self_check/util.py

```diff
@@ -99,6 +99,49 @@
     sqliteDialect = None
     SQLITE_TYPES = {}
 
+try:
+    import pybigquery.sqlalchemy_bigquery
+    import pybigquery.sqlalchemy_bigquery as BigQueryDialect
+
+    # Sometimes "pybigquery.sqlalchemy_bigquery" fails to self-register in certain environments, so we do it explicitly.
+    # (see https://stackoverflow.com/questions/53284762/nosuchmoduleerror-cant-load-plugin-sqlalchemy-dialectssnowflake)
```

Comment:
Do we happen to have a reason why? If so, could we add it to this note?

Reply:
It appears to be something specific to how certain sqlalchemy packages are installed in Azure.. I'll add a note :)